### ISSUE TITLE
Frustrated

### ISSUE THREAD

**Issue by prp1277** [prp1277/prp1277.github.io#1]
I've been working on making a website using Github for the last few months. I made the mistake of forking a project with the premise of getting a site up and running in less than 5 minutes without knowing each file's specific function. 

This has many headaches, a deleted website, a recreated website and another website that is prime for deletion. I don't know that I'm even writing this in a place that it will be visible at this point. I'm very new to programming and web development and I'm cursing the fact that I ever found Beautiful-Jekyll and Github pages. 

All I want is a place to write posts as well as share Excel and Power BI documents, am I in the wrong place?

**Comment by spences10** [comment]
Hey @prp1277 you logged an issue or maybe an email with me the other day and Irecl you mentioning this as a bit of a pain point for you, correct?

If you're new to programming in general then maybe the best thing to do, to get a better understanding of how web dev works then I'd suggest taking a look at [freeCodeCamp(🔥)] it really helped me when transitioning from VBA to web-dev

Jekyll is great for what you want to do, I used to have a site in Jekyll but I moved over to Gatsby.js and have never looked back, the documentation on [gatsby.com] is pretty good too 👍

[freecodecamp(🔥)]: https://www.freecodecamp.org/
[gatsby.com]: https://gatsby.com/

**Comment by prp1277** [comment]
Yes, thank you for the response @spences10 . I couldn't figure out where this thread was and had to dig through a slew of "your page did not build" emails trying to find it. I listened to the Toolsday podcast that you mentioned in one of your posts and looked through the documentation of Gatsby and it seems to make a lot more sense than Jekyll's. 

I started the first class in my Data Science program on Wednesday which will have a primary focus in R. I also learned that the LinkedIn Learning (Lynda) courses are offered for free if you sign up for a library card, so between the two I'm starting to become a bit more familiar.

I wish the Excel community was a bit more active outside of /r/excel and Mr. Excel forums. I didn't even know what version control was until I joined Github and that in itself was a revelation. XLTM files and folders full of "MMDDYYY-FileName" have been the norm for me since I started messing around in Excel.

Now that I have a bit more direction I think I'm going to kill my Jekyll GitHub Pages and redo it using Gatsby. 

I really do appreciate the help - all the languages, frameworks, IDEs, and all the other acronyms that get tossed around have been a bit overwhelming.

**Comment by spences10** [comment]
For version control with Excel I presume you have had a look at https://github.com/spences10/VBA-IDE-Code-Export?

It's no been worked on in a long time but is useable
